content/_releases/22/38.md

```diff
@@ -0,0 +1,161 @@
+---
+
+## Improvements
+
+### Credentials Page: Search by component name
+
+In our quest to improve every aspect of our User Interface, we added a search functionality
+to the Credentials page. When you navigate to Integrate - Credentials you can now
+use the Search by component name to find the credential card you need instead of
+scrolling the page to find it.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2238/RN2238-credentials-search-by-name.gif" title="Credentials Page: Search by component name" %}
+
+### WebHook Step: Send sample request
+
+When you configure WebHook component step in your integration flow, you can either
+**Send sample request** or **Add sample manually**. When you select to send sample
+request a WebHook address is provided which you can use to send your example sample.
+
+We changed the wording to highlight that this address to be used **only one-time** to
+send a representative sample which you can use to configure subsequent steps in your
+integration flow.
+
+{% include img.html max-width="60%" url="/assets/img/RN/2238/RN2238-webhook-send-sample-request.png" title="WebHook step: send sample request" %}
+
+### REST-API-V2 component POST and PUT body type
+
+While configuring the REST-API-v2 component step in integration flows to `POST` or
+`PUT` JSON body, we use `application/json` content type but forget to change the
+input field from [Integrator to JSONata mode](/guides/mapping-data#jsonata-mode).
+When the configuration is saved, the HTTP call body is saved with an incorrect
+format resulting in unexpected or erroneous outcome during the flow execution.
+
+To improve the user experience and prevent an accidental JSON data structure pasting
+into the Body field while in an Integrator mode, we changed the User Interface
+behaviour to default into JSONata mode when an `application/json` content type is
+selected for the `POST` and `PUT` HTTP calls.
+
+{% include img.html max-width="60%" url="/assets/img/RN/2238/RN2238-rest-api-v2-jsonata.png" title="REST-API-V2 step: JSONata mode for POST/PUT" %}
+
+### Run-time environment variables
+
+During the integration step execution, components exchange information with the
+platform services and database entities using already defined run-time environment
+variables. We are constantly working to standardise and add new environment variables
+to streamline the information exchange. The following environment were added to
+the run-time:
+
+*   `WEBHOOK_URI` - This environment variable will be available to each step in the flow which exchanges the data with the webhook service of the platform, eg. [WebHook](/components/webhook/) or [Flow-linking](/components/flow-linking/). This environment variable from the Tenant attributes `webhooks_domain` or from the HELM3 environment variable `WEBHOOKS_BASE_URI` if the Tenant record is missing.
+
+*   `ELASTICIO_CONTRACT_ID` - This environment variable will be available during the run-time of every execution including:
```

Comment:
Would be gret to communicate the benefit here.. To my knowledge this maens that the Contract ID will be attached to each exec log - benefit for multicontract Users and Tenant Owners - easier troubleshooting in event of errors etc. ??